homeworks/week2/trang.vu

```diff
@@ -0,0 +1,33 @@
+import java.util.Set;
+import java.util.List;
+import java.util.ArrayList;
+
+  
+// Time complexity is O(n)
+// Space complexity is O(n) 
+
+
+ class Main {
+    public static void main(String[] args) {
+      String myString = "Hello world!";
+      removeDuplicateCharacters(myString.toCharArray());
```

Comment:
You can have more tests cases here.

Testing is very important in software development (even in coding interview), you can see examples from Tin's CR https://github.com/P2PLearning/DSA/pull/3#pullrequestreview-438724864